admin-front/src/shared/hooks/useGetDataFromAPI.tsx

```diff
@@ -0,0 +1,55 @@
+import { useNotify } from 'react-admin';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+interface GetDataFromAPI {
+  path: string;
+  notify: (value: string) => void;
+  queryStr?: string;
+}
+
+const getDataFromAPI = async ({ path, queryStr, notify }: GetDataFromAPI) => {
+  const baseUrl = process.env.REACT_APP_API_SERVER_BASE_URL;
+  const url = queryStr ? baseUrl + path + queryStr : baseUrl + path;
```

Comment:
Small preference towards `baseUrl + path + (queryStr ?? '')`. Easier to see what the impact of `queryStr` being defined or not is then.